open-source-ecosystem/creating-your-own-instance/index.qmd

```diff
@@ -0,0 +1,19 @@
+---
+title: Using VEDA For Your Own Team
+subtitle: VEDA is open source. We encourage you to use VEDA in your own worlds. Here's how.
+---
+
+## VEDA is a suite of open source tools
+Our primary contribution with VEDA was to extend these tools and provide a blueprint on how they can be used together to effectively visualize and analyze Analysis-Ready, Cloud-Optimized geospatial data. Depending on your needs, you might use VEDA as a tool for:
+
+1. Efficient data storage using a cloud-optimized Data Catalog
+2. Backend Data Services to provide easy access to your data organized in STAC
+3. Cloud computing with your own tailored JupyterHub environment
+4. Browser-based visualization and storytelling using the Dashboard's cloud-native tools
+
+See [Software Architecture](open-source-ecosystem/architecture.qmd) for more information on how all of these pieces fit together. 
+
+There are many options for how to use VEDA tools, depending on your needs and existing set of tools, if any.
+
+Reach out to us at veda@uah.edu if you need help navigating the options.
```

Comment:
I think it would come across more professional to create a "Contact" or "Support" page in our docs and referring to that instead of the email address.

Reply:
I agree. We have veda@uah.edu mentioned in a number of places. We should come back after the support page is in and direct them there.